test/index.test.js

```diff
@@ -1,67 +1,68 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
+const assert = require('node:assert')
 const { Client } = require('@elastic/elasticsearch')
 const Fastify = require('fastify')
 const fastifyElasticsearch = require('..')
 const isElasticsearchClient = require('..').isElasticsearchClient
 
 test('with reachable cluster', async t => {
   const fastify = Fastify()
-  t.teardown(() => fastify.close())
+  t.after(() => fastify.close())
   fastify.register(fastifyElasticsearch, { node: 'http://localhost:9200' })
 
   await fastify.ready()
-  t.equal(fastify.elastic.name, 'elasticsearch-js')
+  assert.equal(fastify.elastic.name, 'elasticsearch-js')
```

Comment:
you should use t.assert.equal etc. and not directly assert

Reply:
Accessing `t.assert` doesn't work as it ends up giving `TypeError [Error]: Cannot read properties of undefined (reading 'equal')`.
Any idea on how to solve it?

Reply:
Done with 85ac08f2f745b771db67e379280cbed7e5489824, but I'm not able to run tests locally anymore because of the error reported previously.

Reply:
> you should use t.assert.equal etc. and not directly assert

Hey @Uzlopak, thanks for the review 😁 Could you explain the rationale behind using `t.assert` instead of `assert`?

Reply:
btw, 
<img width="1373" alt="image" src="https://github.com/user-attachments/assets/41b9c822-0581-4252-ad1f-fc3cc07f3dfb">
I've just tested, and the context assert is not present in Node 20.12 but is present in 20.17.  
It is also not exposed in Node 18.  
I suggest using `import assert from 'node:assert'` if we want to maintain CI compatibility for Node 18, 20, etc.

Reply:
I see that [Node 18 was removed from `plugins-ci-elasticsearch.yml`](https://github.com/fastify/workflows/commit/8dccd9d08455af9fbd594dec11d2bbe83f4e088d#diff-8932628cd606b9c9584872ed40e76b06df045f3286d90316e52866e4ca0189daR29) few months back.
Maybe we should move from the `v4.2.0` to a more recent one? [`v5.0.0`](https://github.com/fastify/workflows/releases/tag/v5.0.0) seems to contain the Node 18 removal.

Reply:
Opened https://github.com/fastify/fastify-elasticsearch/pull/118 to address the `plugins-ci-elasticsearch.yml` reference update.

Reply:
The difference between `t.assert` and `assert` is that `t.assert` works with `t.plan` for assertion counting.
I think in this case does not make any difference.